pwa/src/templates/templateParts/header/HeaderTemplate.tsx

```diff
@@ -222,66 +221,38 @@ export const HeaderTemplate: React.FC<HeaderTemplateProps> = ({ layoutClassName
           </section>
         </Container>
       )}
-      {pathname !== "/" &&
-        ((process.env.GATSBY_USE_GITHUB_REPOSITORY_NAME_AS_PATH_PREFIX === "true" &&
-          pathname !== `/${process.env.GATSBY_GITHUB_REPOSITORY_NAME}/`) ||
-          process.env.GATSBY_USE_GITHUB_REPOSITORY_NAME_AS_PATH_PREFIX === "false") && (
-          <Container layoutClassName={styles.breadcrumbsContainer}>
-            {process.env.GATSBY_ARROW_BREADCRUMBS === "true" && (
-              <BreadcrumbNav className={styles.breadcrumbs} label={t("Breadcrumbs")} appearance="arrows">
-                {translatedCrumbs.map((crumb: any, idx: number) => {
-                  if (crumbs.length !== idx + 1) {
-                    return (
-                      <BreadcrumbNavLink
-                        className={styles.breadcrumbNavLink}
-                        key={idx}
-                        onClick={(e: any) => handleBreadcrumbClick(e, crumb.pathname)}
-                        href=""
-                      >
-                        {crumb.crumbLabel}
-                      </BreadcrumbNavLink>
-                    );
-                  }
-                  return (
-                    <BreadcrumbNavLink key={idx} className={styles.breadcrumbDisabled} current disabled href="">
+      {isHomepage(pathname) && (
```

Comment:
I think we want to show the breadcrumbs when it's **not** the homepage. Right now, reading this, it would suggest that we're showing it when it **is** the homepage.

I would expect `!isHomepage ...`